bedwars-plugin/src/main/java/com/andrei1058/bedwars/arena/Arena.java

```diff
@@ -895,7 +895,7 @@ public void removePlayer(@NotNull Player p, boolean disconnect) {
         /* Remove also the party */
         if (getParty().hasParty(p)) {
             if (getParty().isOwner(p)) {
-                if (status != GameState.restarting) {
+                if (status != GameState.restarting || status != GameState.playing) {
```

Comment:
no. that's not a fix. this is going to break other things. this check was added for safety reasons. requires intensive testing and additional implementations.